views/scan.hbs

```diff
@@ -1,7 +1,7 @@
 <main class="container bg-light clear-header" {{> analytics/results_page_label }}>
   <div class="row jst-cntr scan-results">
 
-    <div class="mw-860 headline-col">
+    <div class="mw-860 headline-col overflow-break">
```

Comment:
On the medium/<800px breakpoint, the `.row` ([L366](https://github.com/mozilla/blurts-server/blob/dashboard-updates/public/css/app.css#L366)) max-width overwrites this change, setting it to same planned value of `600px`, furthering the argument for a break point specific max-width utility class.